test/test-assist/src/main.rs

```diff
@@ -181,6 +188,7 @@ fn main() -> ! {
 
     let fatalops = [
         (AssistOp::BadMemory, badread as fn(u32)),
```

Comment:
Should we rename `BadMemory` to `BadRead`?  It is a little confusing to have `BadMemory` refer to the read test while `BadWrite` is the write test.

Reply:
I think this is a good idea.

Reply:
Will do